File: ClientProject/src/com/company/Animal.java
package com.company;

public class Animal {
    private String id;
    private String name;
    private String kind;
    private char gender;
    private int age;
    private String vaccination;
    private Boolean adopted;
    private String info;
    private String image;
    private String shelterDate;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getInfo() {
        return info;
    }

    public void setInfo(String info) {
        this.info = info;
    }

    public Boolean getAdopted() {
        return adopted;
    }

    public void setAdopted(Boolean adopted) {
        this.adopted = adopted;
    }

    public String getVaccination() {
        return vaccination;
    }

    public void setVaccination(String vaccination) {
        this.vaccination = vaccination;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public char getGender() {
        return gender;
    }

    public void setGender(char gender) {
        this.gender = gender;
    }

    public String getKind() {
        return kind;
    }

    public void setKind(String kind) {
        this.kind = kind;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getShelterDate() {
        return shelterDate;
    }

    public void setShelterDate(String shelterDate) {
        this.shelterDate = shelterDate;
    }
}

File: mobile-client/app/src/main/java/com/example/shelterapp/Animal.kt
package com.example.shelterapp

class Animal {

    private var id: Int = 0
    private var name: String = ""
    private var kind: String = ""
    private var gender: Char = ' '
    private var age: Int = 0
    private var vaccination: String = ""
    private var adopted: Boolean = false
    private var info: String = ""
    private var image: String = ""
    private var shelterDate: String = ""

    fun setId(id: Int){
        this.id = id
    }
    fun getId(): Int{
        return id
    }

    fun setName(name: String){
        this.name = name
    }
    fun getName(): String{
        return name
    }

    fun setKind(kind: String){
        this.kind = kind
    }
    fun getKind(): String{
        return kind
    }

    fun setGender(gender: Char){
        this.gender = gender
    }
    fun getGender(): Char{
        return gender
    }

    fun setAge(age: Int){
        this.age = age
    }
    fun getAge(): Int{
        return age
    }

    fun setVaccination(vaccination: String){
        this.vaccination = vaccination
    }
    fun getVaccination(): String{
        return vaccination
    }

    fun setAdopted(adopted: Boolean){
        this.adopted = adopted
    }
    fun getAdopted(): Boolean{
        return adopted
    }

    fun setInfo(info: String){
        this.info = info
    }
    fun getInfo(): String{
        return info
    }

    fun setImage(image: String){
        this.image = image
    }
    fun getImage(): String{
        return image
    }

    fun setShelterDate(){
        this.shelterDate = shelterDate
    }
    fun getShelterDate(): String{
        return shelterDate
    }



}